Source/Initialization/PlasmaInjector.H

```diff
@@ -131,7 +131,9 @@ public:
 
     InjectorPosition* getInjectorPosition ();
     InjectorDensity*  getInjectorDensity ();
-    InjectorMomentum* getInjectorMomentum ();
+
+    InjectorMomentum* getInjectorMomentumDevice ();
+    InjectorMomentum& getInjectorMomentumHost ();
```

Comment:
Should we make the return types consistent?

Reply:
Do you mean that you would like both functions to return a reference `InjectorMomentum&`?

Reply:
Both `InjectorMomentum*`.

Reply:
Got it, done in [72b2ee2](https://github.com/ECP-WarpX/WarpX/pull/3958/commits/72b2ee27f99ff6489044c8fe84476693da2aff81).